widget/richtext.go

```diff
@@ -549,14 +549,47 @@ func (r *textRenderer) layoutRow(texts []fyne.CanvasObject, align fyne.TextAlign
 		return texts[0].MinSize().Height
 	}
 	height := float32(0)
+	tallestBaseline := float32(0)
+	realign := false
 	for _, text := range texts {
-		size := text.MinSize()
-
+		var size fyne.Size
+		if txt, ok := text.(*canvas.Text); ok {
+			s, base := fyne.CurrentApp().Driver().RenderedTextSize(txt.Text, txt.TextSize, txt.TextStyle)
+			if tallestBaseline == 0 {
+				tallestBaseline = base
+			} else if base != tallestBaseline {
+				tallestBaseline = fyne.Max(tallestBaseline, base)
+				realign = true
+			}
```

Comment:
I think this could be improved to:
```go
if base > tallestBaseline {
    realign = realign || tallestBaseline > 0
    tallestBaseline = base
}
```

Reply:
Yes that is fewer lines of code but it took me a while to figure hot how it works - do you think it is clearer than the original PR?

Reply:
Hmm, no in that form I think is not very clear, maybe this is clearer? 
```go
if base > tallestBaseline {
    if tallestBaseline > 0 {
        realign = true
    }
    tallestBaseline = base
}
```